src/app/directive/components/directive-edit/test-utils.ts

```diff
@@ -52,13 +54,13 @@ export class MockDirectiveRepo {
         },
         status: 404,
         statusText: 'Steward account not found',
-      }) as any as Directive;
+      }) as any as DirectiveData;
       // Simulate returning an unexpected error from the HttpClient
     }
     const testDirectiveId = '39b2a5fa-3508-4030-91b6-21dc6ec7a1ab';
-    const newDirective: Directive = {
+    const newDirective: DirectiveData = {
       directiveId: testDirectiveId,
-      archiveId: directive.archiveId,
+      archiveId: parseInt(directive.archiveId.toString()),
```

Comment:
Why do you have to cast it to a string and then an int?

Reply:
One issue I ran into was that Stela actually expects the `archiveId` to be a string instead of a number. I'm not sure if that's intended, since in the current Permanent API archiveIds are always numbers.
<details>
<summary>Specific explanation of the cast</summary>
In order to not break the current directive components tests right now, I set the `archiveId` type on the Directive model to be a `number | string` to match the usage Stela expects as well as the usage of `ArchiveVO.archiveId` we use in the web-app right now. I didn't want the behavior of this test to change from updating types, so I manually cast the `archiveId` to a number by using `parseInt`... which always expects a string. Since this `archiveId` could either be a string or number, I have to first cast it to a string to ensure it *is* a string before then casting it back into a number.
</details>

I think it might be best just to update the types and tests to match the types that Stela expects, but I wasn't sure I wanted massive updates to these tests to be part of this PR.

Reply:
Thank you! That makes sense. I remember some conversation about stela expecting ids to be strings. I think the idea is that new stela-specific ids will be uuids, which are strings, but I don't remember exactly why we made `archiveId`s strings since they're always ints at the moment.

In any case, I agree with you that big test changes don't belong in this PR.